ui/apps/platform/src/Containers/NetworkGraph/simulation/NetworkPolicySimulatorSidePanel.tsx

```diff
@@ -58,7 +64,81 @@ function NetworkPolicySimulatorSidePanel() {
                     id={tabs.SIMULATE_NETWORK_POLICIES}
                     hidden={activeKeyTab !== tabs.SIMULATE_NETWORK_POLICIES}
                 >
-                    <div>Simulate network policies</div>
+                    <div className="pf-u-p-lg">
+                        <Stack hasGutter>
+                            <StackItem>
+                                <Stack hasGutter>
+                                    <StackItem>
+                                        <TextContent>
+                                            <Text
+                                                component={TextVariants.h1}
+                                                className="pf-u-font-size-lg"
+                                            >
```

Comment:
change component used to `<h2>` for accessibility

```suggestion
                                            <Text
                                                component={TextVariants.h2}
                                                className="pf-u-font-size-lg"
                                            >
```